.github/workflows/asset_compilation.yml

```
@@ -0,0 +1,133 @@
+on:
+  push:
+    branches:
+      - '*'
+      - '**/*'
+
+name: Precompile any assets that have changed and push them to S3
+
+concurrency:
+  group: ${{ github.ref }}-asset-compilation
+  cancel-in-progress: true
+
+jobs:
+  compile:
+    name: Compile
+
+    strategy:
+      matrix:
+        client_group: [load1, load2, load3, load4, load5, load6, load7, load8]
+
+    runs-on: ubuntu-20.04
+    container: ruby:2.7.5-alpine3.13
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install GNU tar (required by cache step and checksumming)
+      run: |
+        apk add --no-cache tar
+
+    # According to https://www.jessesquires.com/blog/2021/08/23/caching-bundler-on-github-actions/
+    # this is fragile and failure prone, but the step they recommend using instead (ruby/ruby-setup)
+    # isn't supported on Alpine.
+    - name: cache gems
+      uses: actions/cache@v2
+      with:
+        path: |
+          vendor/bundle
+          /usr/local/bundle/
+        key: ${{ runner.os }}-gem-bundler-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gem-bundler-
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
+
+    - name: Set up dependencies
+      env:
+        DATABASE_APP_DB_TEST: boston_hmis_test
+        WAREHOUSE_DATABASE_DB_TEST: warehouse_test
+        HEALTH_DATABASE_DB_TEST: health_test
+        DATABASE_DB_TEST: boston_cas_test
+        REPORTING_DATABASE_DB_TEST: test_boston_reporting
```

These can probably just be:
```suggestion
        DATABASE_DB_TEST: cas_test
        REPORTING_DATABASE_DB_TEST: reporting_test
```